ui-admin/src/forms/designer/split/SplitFormDesigner.tsx

```diff
@@ -16,81 +22,78 @@ export const SplitFormDesigner = ({ content, onChange, currentLanguage, supporte
     content: FormContent, onChange: (newContent: FormContent) => void,
     currentLanguage: PortalEnvironmentLanguage, supportedLanguages: PortalEnvironmentLanguage[]
 }) => {
+  const HIDE_TABLE_OF_CONTENTS_KEY = 'formDesigner.hideTableOfContents'
+
   const [currentPageNo, setCurrentPageNo] = useState(0)
-  const [currentPage, setCurrentPage] = useState(content.pages[currentPageNo])
+  const [hideTableOfContents, setHideTableOfContents] = useState(
+    localStorage.getItem(HIDE_TABLE_OF_CONTENTS_KEY) === 'true')
+  const [searchParams, setSearchParams] = useSearchParams()
+  const selectedElementPath = searchParams.get('selectedElementPath') ?? 'pages'
 
-  const handlePageChange = (direction: 'next' | 'previous') => {
-    if (direction === 'next') {
-      setCurrentPageNo(currentPageNo + 1)
-      setCurrentPage(content.pages[currentPageNo + 1])
-    } else {
-      setCurrentPageNo(currentPageNo - 1)
-      setCurrentPage(content.pages[currentPageNo - 1])
-    }
-    handleScrollToTop()
+  const setSelectedElementPath = (path: string) => {
+    searchParams.set('selectedElementPath', path)
+    setSearchParams(searchParams)
+    scrollToElement(path, setCurrentPageNo)
   }
 
-  const handleScrollToTop = () => {
-    window.scrollTo(0, 0)
-  }
+  useEffect(() => {
+    if (selectedElementPath && selectedElementPath !== 'pages') {
+      scrollToElement(selectedElementPath, setCurrentPageNo)
+    }
+  }, [])
 
-  return <div className="mt-3 container w-100">
-    <div className="d-flex">
-      {renderNewElementButton(content, onChange, -1, currentPageNo, 'question')}
-      {renderNewElementButton(content, onChange, -1, currentPageNo, 'panel')}
-    </div>
-    <>
-      {currentPage && currentPage.elements && currentPage.elements.map((element, elementIndex) => (
-        <div key={elementIndex} className="container">
-          <SplitFormElementDesigner currentPageNo={currentPageNo}
-            elementIndex={elementIndex} editedContent={content}
-            element={content.pages[currentPageNo].elements[elementIndex]}
-            currentLanguage={currentLanguage} supportedLanguages={supportedLanguages}
-            onChange={onChange}/>
-          <div className="d-flex">
-            {renderNewElementButton(content, onChange, elementIndex, currentPageNo, 'question')}
-            {renderNewElementButton(content, onChange, elementIndex, currentPageNo, 'panel')}
+  return <div className="container-fluid overflow-scroll">
+    <div className="row w-100 mx-0">
+      <div style={{ overflowY: 'scroll' }}
+        className={classNames('px-0 border-start border-end bg-white', hideTableOfContents ? 'd-none' : 'col-3')}>
```

Comment:
```suggestion
        className={classNames('px-0 bg-white', hideTableOfContents ? 'd-none' : 'col-3')}>
```
UX nit: I think it looks cleaner without borders

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/5e55b978-34f5-41a9-9d83-ec34d9545ccb">